build.sbt

```diff
@@ -203,6 +207,22 @@ lazy val monix = crossProject(JSPlatform, JVMPlatform)
 lazy val monixJVM = monix.jvm
 lazy val monixJS  = monix.js
 
+lazy val monixBio = crossProject(JSPlatform, JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("modules/monixBio"))
+  .dependsOn(core, framework % "test->compile")
+  .configure(WeaverPlugin.profile)
+  .settings(WeaverPlugin.simpleLayout)
+  .settings(
+    libraryDependencies ++= Seq(
+      "io.monix" %% "monix-bio" % "1.0.0"
```

Comment:
There's a mistake here which causes the build to fail. I'll let you find what it is 😄

Reply:
🤦 I'm guessing I need `%%%` to get the Scala.js source, too.